cmd/juju/subnet/remove.go

```diff
@@ -24,7 +23,7 @@ func NewRemoveCommand() modelcmd.ModelCommand {
 type RemoveCommand struct {
 	SubnetCommandBase
 
-	CIDR names.SubnetTag
+	CIDR string
```

Comment:
Swapping names.SubnetTag for string so that juju compiles and the tests run.  This command needs audit and work, per the already created trello card.